tests/test_prepare.py

```diff
@@ -28,12 +28,14 @@ def test_prepare(tmpdir, monkeypatch):
         "makeme321": 123,
         "makemenegative": 0.0001,
     }
+    url1 = "https://dist.valohai.com/valohai-utils-tests/Example.svg"
+    url2 = "https://dist.valohai.com/valohai-utils-tests/sharktavern.jpg"
```

Comment:
how... a glass of water, underwater!?

Reply:
https://en.wikipedia.org/wiki/Heavy_water duh

Reply:
sharks like their water heavy 🪨 :weight_lifting: